spring-boot-starter/starter/src/main/java/org/camunda/bpm/spring/boot/starter/actuator/ProcessEngineHealthIndicator.java

```diff
@@ -26,7 +26,7 @@ public class ProcessEngineHealthIndicator extends AbstractHealthIndicator {
   private final ProcessEngine processEngine;
 
   public ProcessEngineHealthIndicator(ProcessEngine processEngine) {
-    Assert.notNull(processEngine);
```

Comment:
FYI: This line is the reason I started looking into this.

After trying to upgrade our application to Spring Boot 3.2-RC1 it failed here with a NoSuchMethodError:

```
Caused by: java.lang.NoSuchMethodError: 'void org.springframework.util.Assert.notNull(java.lang.Object)'
	at org.camunda.bpm.spring.boot.starter.actuator.ProcessEngineHealthIndicator.<init>(ProcessEngineHealthIndicator.java:29)
```

Reply:
I've run into the same issue, I wanted to contribute this myself, thank you for doing it. Waiting for 7.21.0 🥳